llvm/lib/ObjectYAML/ELFYAML.cpp

```diff
@@ -620,6 +620,15 @@ void ScalarBitSetTraits<ELFYAML::ELF_EF>::bitset(IO &IO,
       BCase(EF_AMDGPU_FEATURE_XNACK_V3);
       BCase(EF_AMDGPU_FEATURE_SRAMECC_V3);
       break;
+    case ELF::ELFABIVERSION_AMDGPU_HSA_V6:
```

Comment:
I think we'd need to add a test for this?

Reply:
elf-headers.test already covers it